src/Quiz.js

```diff
@@ -217,52 +177,45 @@ class Quiz extends Component {
 					{/* <i className="fa-solid fa-arrow-rotate-left" /> */}
 					<i className="fa-solid arrow-left" />
 					Finished all questions!
-					<button className="Quiz-restart-btn" onClick={this.resetQuestions}>
+					<button className="Quiz-restart-btn" onClick={() => resetGame(false)}>
```

Comment:
Pro-tip: Defining anonymous arrow functions as callbacks actually creates a new function on every render 😮 . It's not a bug, but it's a slight performance issue if you only do it this way. I'd split this out into a `useCallback` and pass the callback in without an anonymous arrow function.

Reply:
Thanks for the tip! I was a bit confused how to use `useCallback` here so ended up just defining separate named functions for each of the true/false cases. (Will keep `useCallback` in mind though)